packages/atomic/src/components/search/atomic-generated-answer/styles/copy-button.pcss

```diff
@@ -0,0 +1,16 @@
+[part='copy-button'] {
+  .icon-container {
+    atomic-icon {
+      &:hover {
```

Comment:
In the mockups, I see that the icon should go blue on hover.

Is the change voluntary?

Reply:
@fbeaudoincoveooh your are right I got confused when it goes green when it's copied, I will fix this :)